src/org/loklak/harvester/YoutubeScraper.java

```diff
@@ -205,9 +203,14 @@ public static JSONObject parseVideo(final BufferedReader br) throws IOException
         return json;
     }
     
-    private static long parseNumber(String n) throws NumberFormatException {
-        String withoutSpaces = CharMatcher.WHITESPACE.removeFrom(n);
-        return Long.parseLong(withoutSpaces);
+    private static String parseValue(String n) {
+        String number_value = CharMatcher.WHITESPACE.removeFrom(n);
+        Pattern number_format = Pattern.compile("((\\d)\\d{0,2})(,?\\d{3})*(\\.[\\d]+)?([MBK]?)");
```

Comment:
> 123,45 isn't valid number

exactly, but it will get accepted by the regex as two discrete matches - `123` and `45`.